src/Home.tsx

```diff
@@ -236,10 +231,7 @@ const Home = ({
 												</thead>
 												<tbody>
 													{data.gateways && data.gateways[activeNet].map((node: string[]) => (
-														<tr key={node[0]}>
-															<td>
-																{node[0]}
-															</td>
+														<tr>
```

Comment:
add `key={node[1]}`

Reply:
I'm not sure why we have the `key` here. It doesn't make much sense to me.

Reply:
Yes, it's done for react, otherwise it shows warning in the console

Reply:
Huh. Surprising. Especially since I don't see it in the resulting HTML and it's just a table row. OK, fixed.